modules/custom/activity_logger/activity_logger.module

```diff
@@ -108,7 +108,7 @@ function activity_logger_form_message_template_form_alter(&$form, FormStateInter
     '#title' => t('The entity condition that are affected for this message'),
     '#description' => t('Select a entity condition for this message.'),
     '#default_value' => $activity_entity_condition_value,
-    '#options' => $activity_entity_condition_options,
+    '#options' => ['' => '- none -'] + $activity_entity_condition_options,
```

Comment:
Can we use `'#empty_option' => t('- none -'),` instead?

Reply:
@rollki Fixed